JSQMessagesViewController/Categories/UIViewController+JSQMessages.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Created by Jesse Squires
+//  http://www.jessesquires.com
+//
+//
+//  Documentation
+//  http://cocoadocs.org/docsets/JSQMessagesViewController
+//
+//
+//  GitHub
+//  https://github.com/jessesquires/JSQMessagesViewController
+//
+//
+//  License
+//  Copyright (c) 2014 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
+
+#import <UIKit/UIKit.h>
+
+@interface UIViewController (JSQMessages)
```

Comment:
Do you intend for this to be public? And clients can import as-needed?

I was going to suggest this should be internal, but maybe it's better to allow clients to opt-in.